src/main/kotlin/me/gserv/fabrikommander/commands/RtpCommand.kt

```diff
@@ -0,0 +1,80 @@
+package me.gserv.fabrikommander.commands
+
+import me.gserv.fabrikommander.utils.Context
+import me.gserv.fabrikommander.utils.Dispatcher
+import me.gserv.fabrikommander.utils.gold
+import me.gserv.fabrikommander.utils.gray
+import me.gserv.fabrikommander.utils.yellow
+import me.gserv.fabrikommander.utils.plus
+import me.gserv.fabrikommander.utils.aqua
+import net.minecraft.server.command.CommandManager
+import net.minecraft.util.Identifier
+import net.minecraft.server.world.ServerWorld
+import net.minecraft.util.registry.Registry
+import net.minecraft.util.registry.RegistryKey
+import net.minecraft.util.math.BlockPos
+
+class RtpCommand(val dispatcher: Dispatcher) {
+    private val rtpRange = -5000..5000
+
+    fun register() {
+        dispatcher.register(
+            CommandManager.literal("rtp")
+                .executes { rtpCommand(it) }
+        )
+        dispatcher.register(
+            CommandManager.literal("wild")
+                .executes { rtpCommand(it) }
+        )
+    }
+    
+    private fun generateCoordinates(world: ServerWorld?): List<Int> {
+        val x = rtpRange.random()
+        val z = rtpRange.random()
+        val y = getHighestBlock(world, x, z)
+
+        if (world?.isWater(BlockPos(x, y, z)) == true) {
+            return generateCoordinates(world)
+        }
+
+        return listOf(x, y + 1, z)
+    }
+
+    private fun getHighestBlock(world: ServerWorld?, x: Int, z: Int): Int {
+        val heightLimit = world!!.heightLimit
```

Comment:
The auto build check keeps failing, saying that heightLimit cannot be found, but it works when I build it on my computer. From what I can tell, the file I have is the same as this one except for the few /back related lines.